tests/network/vmi_multus.go

```diff
@@ -834,6 +867,29 @@ func cloudInitNetworkDataWithStaticIPsByDevice(deviceName, ipAddress string) str
 	return networkData
 }
 
+func cloudInitConfigDriveNetworkData(NICName, NADName, MACAddress, ipAddress, netmask string) string {
+	networkData := configdrive.NewNetworkData(
+		configdrive.WithLink(
+			configdrive.Link{
+				Id:                 NICName,
```

Comment:
Why the `NICName` (I think it is specNetworkName and not nic name) is passed, I guess you nic some unique id, not necessary the nic name?
Is the id mandatory? If yes, I would generate some random link and network id or at least call the variables "networkId` and `linkId`.

Reply:
Changed the parameter names to `linkId` and `networkId`.
In the [schema](https://docs.openstack.org/nova/latest/_downloads/9119ca7ac90aa2990e762c08baea3a36/network_data.json) they are specified as just strings (without limitations).

The reason to pass the network/interface name and NAD name as `linkId` / `networkId` is to be able to identify them in the cloud-init logs.